bin/autodeploy_devxpi.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# A script to automatically build, and upload to AMO the latest version of the loop
+# add-on xpi, including the latest localisations from loop-client-l10n.
+
+# This is designed to be run from a crontab somewhere.
+
+# WARNING: Running this script will modify your repositories.
+
+# The script expectes the following repository setup:
+#   ${BASEDIR}/loop - A clone of https://github.com/mozilla/loop
+#   ${BASEDIR}/loop-client-l10n - A clone of https://github.com/mozilla/loop-client-l10n
+
+# This script expects JPM_API_KEY and JPM_API_SECRET to be present in the environment.
+# See the upload_xpi command in the Makefile.
```

Comment:
I'd suggest changing all of this from a comment to [here document](https://bash.cyberciti.biz/guide/Here_documents) and echoing it out as part of the usage message.  It would also be nice if it describes how your repo is changed, and potentially makes a recommendation for how to use it (eg make a scratch repo, use it there, then delete that repo, or maybe do it in your regular repo and then just do a 'git reset --hard master' afterwards or something)...